admin_manual/enterprise/firewall/file_firewall.rst

```diff
@@ -3,7 +3,7 @@ File Firewall
 =============
 
 The File Firewall GUI enables you to create and manage firewall rule sets from 
-your ownCloud admin page. The File Firewall gives you finer-grained control of 
+your ownCloud admin page, Security section. The File Firewall gives you finer-grained control of 
```

Comment:
One, minor, suggestion: "your ownCloud admin page's, Security section." Otherwise, it looks fine to me.